script/cibuild.sh

```diff
@@ -49,10 +49,24 @@ export GITTEST_REMOTE_SSH_KEY="$HOME/.ssh/id_rsa"
 export GITTEST_REMOTE_SSH_PUBKEY="$HOME/.ssh/id_rsa.pub"
 export GITTEST_REMOTE_SSH_PASSPHRASE=""
 
+
+# Can we ask Travis to cache this file?
+curl https://github.com/ethomson/poxyproxy/releases/download/v0.1.0/poxyproxy-0.1.0.jar >poxyproxy.jar || exit $?
```

Comment:
You could put it somewhere like `vendor/poxyproxy/poxyproxy.jar` and then add this to the travis config:

```
cache:
  directories:
  - vendor/poxyproxy
```